converters/toZigbee.js

```diff
@@ -1163,6 +1163,45 @@ const converters = {
             entity.commandResponse('ssIasAce', 'panelStatusChanged', payload);
         },
     },
+    ballast_config: {
+        key: ['ballast_config'],
+        convertSet: async (entity, key, value, meta) => {
+            for (const [attrName, attrValue] of Object.entries(value)) {
+                const attributes = {};
+                attributes[attrName] = attrValue;
+                await entity.write('lightingBallastCfg', attributes);
```

Comment:
Can you also convert this to snake_case? You can copy the following function to do the conversion: https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/util/utils.js#L157